pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -673,10 +674,7 @@ func (m *kubeGenericRuntimeManager) doPodResizeAction(pod *v1.Pod, podStatus *ku
 		case v1.ResourceMemory:
 			err = pcm.SetPodCgroupConfig(pod, rName, podResources)
 		}
-		if err != nil {
-			klog.ErrorS(err, "Failed to set cgroup config", "resource", rName, "pod", pod.Name)
-		}
-		return err
+		return fmt.Errorf("failed to set cgroup config for %s of pod %s: %w", rName, format.Pod(pod), err)
```

Comment:
This is unconditionally returning an error for this function.

Reply:
I'm sorry I forgot to run tests after updating.

Reply:
I believe the presubmit job is run when these files are touched, but I overlooked checking it on this instance, and missed the bug in code review as well - my bad (note to self: dont review code with tired eyes)
Imho this may be a good time to make that presubmit job visible.

Reply:
I believe the e2e tests presubmit job runs when these files are touched. But I overlooked checking for the outcome on this occasion and missed the bug in the review as well, my bad.
Imho this is a good time to make pull-kubernetes-e2e-inplace-pod-resize-containerd-main-v2 visible in PRs.

Reply:
PR https://github.com/kubernetes/test-infra/pull/33781